webservice/src/UserLogic.jsx

```diff
@@ -284,34 +223,29 @@ export default class UserLogic extends React.Component {
   }
 
   /**
-   * Function to stop and delete the current selected CRD instance
+   * Function to stop and delete an instance
    */
-  stopCRDinstanceAdmin() {
-    const { selectedInstance, instanceLabsAdmin } = this.state;
-    if (!selectedInstance) {
-      Toastr.info('No lab to stop has been selected');
-      return;
-    }
-    if (!instanceLabsAdmin.has(selectedInstance)) {
-      Toastr.info(`The \`${selectedInstance}\` lab is not running`);
+  stopCRDinstanceAdmin(instanceName) {
+    const { instanceLabsAdmin } = this.state;
+    if (!instanceLabsAdmin.has(instanceName)) {
+      Toastr.info(`The \`${instanceName}\` lab is not running`);
       return;
     }
     this.apiManager
       .deleteCRDinstance(
-        selectedInstance,
-        instanceLabsAdmin.get(selectedInstance).studNamespace
+        instanceName,
+        instanceLabsAdmin.get(instanceName).studNamespace
```

Comment:
Here the `studNamespace` property is used, but it is no longer set. Hence, the deletion of a laboratory from the professor view fails.